packages/@expo/cli/src/start/server/metro/inspector-proxy/device.ts

```diff
@@ -65,7 +65,9 @@ export function createInspectorDeviceClass(
       if (oldDebugger) {
         oldDebugger.socket.removeAllListeners();
         this._deviceSocket.close();
-        newDevice.handleDebuggerConnection(oldDebugger.socket, oldDebugger.pageId);
+        newDevice.handleDebuggerConnection(oldDebugger.socket, oldDebugger.pageId, {
+          userAgent: oldDebugger.userAgent,
+        });
```

Comment:
Not 100% set on the `userAgent` name, it works like a user agent, but its not implemented as such (so probably something needs to change on that -- not sure what yet)

Reply:
It's the name used by @react-native/dev-middleware so we might need to live with it